src/net.h

```diff
@@ -1197,19 +1197,25 @@ class CConnman
 
     std::unique_ptr<CSemaphore> semOutbound;
     std::unique_ptr<CSemaphore> semAddnode;
-    int m_max_automatic_connections;
-
-    // How many full-relay (tx, block, addr) outbound peers we want
```

Comment:
did you drop the doc for  `m_max_outbound_full_relay` and `m_max_outbound_block_relay` on purpose?